include/mbgl/storage/default_file_source.hpp

```diff
@@ -18,8 +20,13 @@ class DefaultFileSource : public FileSource {
 
     std::unique_ptr<FileRequest> request(const Resource&, Callback) override;
 
+    // For testing purposes only.
```

Comment:
Can we make this private: https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#private-class-members